test/LibraryManager.IntegrationTest/Microsoft.Web.LibraryManager.IntegrationTest.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove=".editorconfig" />
+    <None Remove="TestSolution\testFile.txt" />
+    <None Remove="TestSolution\WebApplication1\Web.Debug.config" />
+    <None Remove="TestSolution\WebApplication1\Web.Release.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Test.Apex.VisualStudio" Version="15.8.27703" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.2.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.2.1" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.23" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="TestSolution\**\*">
```

Comment:
Instead of EmbeddedResource, if you use `[DeploymentItem]` you'll want to change this to

    <Content Include="TestSolution\**\*">
        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </Content>

Reply:
Will do

Reply:
Why is this changing back to EmbeddedResource?  I thought we had it working with Content?